#StartDetection.srv
string mode
string class_name
bool startdetect
---
float64 xmin
float64 xmax
float64 ymin
float64 ymax
string class_name
bool success
string message